src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -43,6 +43,7 @@ public AdminController(WebScriptHostManager scriptHostManager, WebHostSettings w
         }
 
         [HttpPost]
+        [RequiresRunningHost]
```

Comment:
I could only identify 4 endpoints that actually require the host to be up and running. Did I miss any? Previously we were requiring the host to be running for ALL endpoints except a single exclusion. I'm inverting that logic, so now endpoints need to be explicitly marked.